Dockerfile

```diff
@@ -43,6 +43,9 @@ EXPOSE 5000
 
 RUN useradd --create-home redash
 
+# DB2 for IBM i support - Official IBM repo
+RUN curl https://public.dhe.ibm.com/software/ibmi/products/odbc/debs/dists/1.1.0/ibmi-acs-1.1.0.list | tee /etc/apt/sources.list.d/ibmi-acs-1.1.0.list
```

Comment:
This surely does not work on all platforms, I assume a check for linux/amd64 at least